interfaces/Portalicious/README.md

```diff
@@ -1,30 +1,46 @@
-# Portalicious
+# Portal(icious)
 
-This project was generated with [Angular CLI](https://github.com/angular/angular-cli).
+The front-end for the _Humanitarian Organization_ web portal, where users can interact with the 121 Platform.
 
-## Development server
+## Configuration
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The application will automatically reload if you change any of the source files.
+Set all ENV-variables in a local `.env`-file, based on: [`.env.example`](.env.example)-file. Each allowed/required value is explained in the example file.
```

Comment:
```suggestion
Set all ENV-variables in a local `.env`-file, based on: [`.env.example`](.env.example)-file. Each allowed/required value is explained in the example file. Do this quickly with `cp .env.example .env`
```

Reply:
Hmm... I guess I've copy-pasted some sentences around too much that this "fell off the wagon". ;)

I've tried to automate this:
https://github.com/global-121/121-platform/commit/3b9a8d71de8440ba80083a9da6d7e93d2b57cbff

But it had a bit of a downside as well, so I didn't dare to put it in yet..
I almost created (several) PRs/Issues on that tool's repo. ;)